[1.x] Minor improvements + bump up to 1.x (#18)
Pod-Point/laravel-mail-export: README.md

You can install the package via composer:

For Laravel 9.x

```bash

> composer require pod-point/laravel-mail-export

Author: Yeah same, `README.md` is the same too.

If we would prefer them to be version specifics, we would need to tackle this through another PR.

So far this makes it easier to maintain as we just copy/paste the files across, also hints the devs they should upgrade.

You will be able to specify:

* `enabled`: wether this package is enabled or not. Once installed, it's enabled by default but the `MAIL_EXPORT` environment variable can be used to configure this.
* `disk`: which disk to use by default. `null` will use the default disk from your application filesystem.

> * `path`: the default path you would like to export your mails within a storage disk.

Reviewer: ```suggestion
* `path`: the default path, within the configured disk, where mail will be exported.
```
